plugins/community/stephenh-ts-proto/v1.176.0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.7
+FROM node:20.13.1-alpine3.19 AS build
+WORKDIR /app
+COPY --link package*.json .
+RUN npm ci
+RUN sed -i -e 's|/usr/bin/env node|/nodejs/bin/node|g' /app/node_modules/ts-proto/protoc-gen-ts_proto \
+ && rm -f /app/node_modules/dprint-node/*darwin*.node /app/node_modules/dprint-node/*win32*.node
+
+FROM gcr.io/distroless/nodejs20-debian12:latest@sha256:a69b487c1d156e3aeaab0ffb587f46248c8e891e289081a3c28f7104a69c4515 AS node
+
+FROM gcr.io/distroless/cc-debian12:latest@sha256:e6ae66a5a343d7112167f9117c4e630cfffcd80db44e44302759ec13ddd2d22b AS base
+
+FROM scratch
+COPY --link --from=base / /
```

Comment:
Testing squashing on base image layers based on @DMarby example. This reduces total number of layers in the image from 18 -> 3 (base image, node binary, plugin).